### ISSUE TITLE
Exception when selecting more than 1 node in the graph editor

### ISSUE THREAD

**Issue by hiroki-o** [Unity-Technologies/AssetGraph#107]
**[Original report](https://bitbucket.org/Unity-Technologies/assetbundlegraphtool/issue/72) by Lior Tal (Bitbucket: [liortal](https://bitbucket.org/liortal), GitHub: [liortal](https://github.com/liortal)).**

----------------------------------------

This happens every time i select (by creating a rectangle) around 2 nodes in the graph

```
#!c#

NullReferenceException: Object reference not set to an instance of an object
UnityEngine.AssetBundles.GraphTool.GroupViewController..ctor (UnityEngine.AssetBundles.GraphTool.GroupViewContext ctx) (at Assets/UnityEngine.AssetBundleGraph/Editor/GUI/GroupView/GroupViewController.cs:114)
UnityEngine.AssetBundles.GraphTool.ConnectionGUIEditor.OnInspectorGUI () (at Assets/UnityEngine.AssetBundleGraph/Editor/GUI/ConnectionGUIEditor.cs:29)
UnityEditor.InspectorWindow.DrawEditor (UnityEditor.Editor[] editors, Int32 editorIndex, Boolean rebuildOptimizedGUIBlock, System.Boolean& showImportedObjectBarNext, UnityEngine.Rect& importedObjectBarRect) (at /Users/builduser/buildslave/unity/build/Editor/Mono/Inspector/InspectorWindow.cs:1243)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr)

```

**Comment by hiroki-o** [comment]
**Original comment by Hiroki Omae (Bitbucket: [Hiroki Omae](https://bitbucket.org/Hiroki Omae), ).**

----------------------------------------

Thank you for reporting this bug. I have reproduced the issue.

**Comment by hiroki-o** [comment]
**Original comment by Lior Tal (Bitbucket: [liortal](https://bitbucket.org/liortal), GitHub: [liortal](https://github.com/liortal)).**

----------------------------------------

please note that i also sent a PR for fixing it

**Comment by hiroki-o** [comment]
**Original comment by Hiroki Omae (Bitbucket: [Hiroki Omae](https://bitbucket.org/Hiroki Omae), ).**

----------------------------------------

Fix issue #72

**Comment by hiroki-o** [comment]
**Original comment by Hiroki Omae (Bitbucket: [Hiroki Omae](https://bitbucket.org/Hiroki Omae), ).**

----------------------------------------

Merged in liortal/assetbundlegraphtool/issue_72 (pull request #11)

Fix issue #72

**Comment by hiroki-o** [comment]
**Original comment by Hiroki Omae (Bitbucket: [Hiroki Omae](https://bitbucket.org/Hiroki Omae), ).**

----------------------------------------

PR accepted. Thank you for contributing!
